packages/components/pages/f-checkout/jest.config.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
     },
 
     transformIgnorePatterns: [
-        'node_modules/(?!(lodash-es)/)'
+        'node_modules/(?!(lodash-es|axios)/)'
```

Comment:
More of a question but why was this changed made in f-checkout? Does `f-self-exclusion` relate to checkout?

Reply:
No changes were made from our end to f-checkout. Not sure why this happened.

Reply:
@siggerzz explained on our internal comms that he did this to fix the failing `f-checkout` tests that this PR uncovered